app/helpers/application_helper.rb

```diff
@@ -7,4 +7,8 @@ def user_image_tag(owner)
     email = owner.nil? ? nil : owner.email
     gravatar_image_tag(email)
   end
+
+  def any_minion?
+    !Minion.all.count.zero?
```

Comment:
This could be simplified to: `Minion.any?` 😉 In which case, is it really needed this helper then ?